omero/sysadmins/server-backup-and-restore.txt

```diff
@@ -63,9 +63,8 @@ Understanding backup sources
 OMERO.server has three main backup sources:
 
 1.  PostgreSQL database (assumed to be ``omero_database``)
-2.  OMERO.server binary data store 
-    (:doc:`UNIX/Mac information page <unix/server-binary-repository>` or 
-    :doc:`Windows information page <windows/server-binary-repository>`; 
+2.  OMERO.server
+    :doc:`binary data store <unix/server-binary-repository>`;
     assumed to be ``/OMERO`` or ``C:\OMERO``)
```

Comment:
`C:\OMERO` is something which should also be dropped.  And `:file:` should be used for `/OMERO`.

It might be worth looking at the result of

```
git grep -i "C:\\\\"
```

to find all instances of this.